pkgs/development/libraries/libuv/default.nix

```diff
@@ -54,6 +54,8 @@ stdenv.mkDerivation rec {
 
   nativeBuildInputs = [ automake autoconf libtool pkg-config ];
   buildInputs = lib.optionals stdenv.isDarwin [ ApplicationServices CoreServices ];
+  patches = [ ./3735-fix-musl-libc-dlerror-test-expectation.patch ];
```

Comment:
Since this patch is available on github I would use `fetchpatch` to get it.

Also for every patch applied in nixpkgs we should have commit explaining why it's there and what is required for it to be removed. IT can be something short like:

```
# Fix tests for statically linked variant
# upstream PR is ....
```

2nd line is almost unnecessary if you use fetchpatch.